webui/react/src/App.tsx

```diff
@@ -159,9 +160,11 @@ const App: React.FC = () => {
     <HelmetProvider>
       <StoreProvider>
         <AgentsProvider>
-          <DndProvider backend={HTML5Backend}>
-            <AppView />
-          </DndProvider>
+          <WorkspacesProvider>
```

Comment:
All the nested providers here  are starting to get a little silly, especially since  I believe the plan is to keep adding more of them. Maybe it would be good to extract them out to a single `Providers.tsx` file.

Reply:
I'll combine the store providers into one export in the `stores` folder.